test/factories/config.rb

```diff
@@ -2,6 +2,5 @@
   factory :config do
     config_key { :insurance }
     config_value { { options: ['DC Medicaid', 'No insurance', "Don't know"] } }
-    created_by { FactoryBot.create(:user) }
```

Comment:
I get why we're untying this now, but its temporary until the User model migrates, right? If so, I'd rather we comment out & tag to update with when we move User. Thoughts?

Reply:
This is a great flag - Config (and Event) are always machine-generated, and so the created_by is always going to be whatever user triggers the autosetup. In mongo world we were userstamping this mostly so we could get modified_by ids, but in paper trail world that gets delegated to the paper trail entry, and so created_by/modified_by fields on the Config model itself get dropped.